frontend/packages/integration-tests-cypress/support/i18n.ts

```diff
@@ -1,13 +1,54 @@
 import { expect } from 'chai';
+import { DETAILS_PAGE, LIST_PAGE } from '../views/common';
 
 declare global {
   namespace Cypress {
     interface Chainable<Subject> {
       isPseudoLocalized(): Chainable<Element>;
+      testI18n(kind: string, pageType: string): Chainable<Element>;
     }
   }
 }
 
+Cypress.Commands.add('testI18n', (kind: string, pageType: string) => {
+  const tableColumnHeaders =
+    'th > .pf-c-table__button > .pf-c-table__button-content > .pf-c-table__text';
+  const detailSectionHeadings = '[data-test-section-heading]';
+  const detailLabels = 'dt';
+
+  const selectors = [];
+  const testIDs = [];
+
+  cy.location().then((loc) => {
+    const params = new URLSearchParams(loc.search);
+    params.set('pseudolocalization', 'true');
+    params.set('lng', 'en');
+    const pseudoLocUrl = `${loc.pathname}?${params.toString()}`;
+
+    cy.log(`Testing I18n for ${kind} ${pageType}: ${pseudoLocUrl}`);
+
+    cy.visit(pseudoLocUrl);
+
+    if (pageType === LIST_PAGE) {
```

Comment:
I'd remove the concept of page type here and just take in the selectors or test IDs to check as parameters. It's better if this helper has not knowledge of specific pages IMO. We can have constants for different page types.

Reply:
fixed.